third_party/webrtc/modules/desktop_capture/mac/screen_capturer_mac.mm

```diff
@@ -20,87 +20,6 @@
 #include "rtc_base/trace_event.h"
```

Comment:
Since we don't support desktop capture whatsoever on Cobalt --
perhaps just setting `rtc_desktop_capture_supported` in args.gn
to false would make modifications to this file unnecessary, and
save some binary size as side benefit?

[rtc_desktop_capture_supported](https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/main:third_party/webrtc/webrtc.gni;l=332-334?q=rtc_desktop_capture_supported&ss=h%2Flbshell-internal%2Fcobalt_src%2F%2B%2Frefs%2Fheads%2Fmain:third_party%2Fwebrtc%2F)